core/src/main/scala/chisel3/DataMirror.scala

```diff
@@ -0,0 +1,301 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import chisel3.internal.firrtl._
+import chisel3.internal._
+import chisel3.experimental.BaseModule
+import scala.reflect.ClassTag
+
+private[chisel3] object DataMirror {
```

Comment:
so is this PR privatizing all of dataMirrors functions?

Reply:
No, I'm exposing it as a public val in the experimental package.

Reply:
Wow good catch, it isn't! Going to make it public.